nsv13/code/__HELPERS/overmap.dm

```diff
@@ -1,12 +1,12 @@
 /atom/proc/get_overmap() //Helper proc to get the overmap ship representing a given area.
 	RETURN_TYPE(/obj/structure/overmap)
-	if(!z)
+	if(loc && istype(loc, /obj/structure/overmap))
+		return loc
+	if(!loc?.z)
```

Comment:
if loc is null you'll probably want it to runtime
```suggestion
	if(isovermap(loc))
		return loc
	if(!loc.z)
```

Reply:
> if loc is null you'll probably _want_ it to runtime, as something else has gone catastrophically wrong

The problem is there's plenty of things in nullspace calling get_overmap() for some reason and I'm gonna have to fix all of them then...

Reply:
Better to have the runtime log than not though right? Null loc is (almost) always unexpected behavior